context/app/static/js/components/Detail/MetadataTable/MetadataTable.jsx

```diff
@@ -55,6 +88,7 @@ function MetadataTable(props) {
 
 MetadataTable.propTypes = {
   metadata: PropTypes.objectOf(PropTypes.string).isRequired,
+  display_doi: PropTypes.string.isRequired,
 };
 
-export default MetadataTable;
+export default React.memo(MetadataTable);
```

Comment:
I see that additional computation is being done in this component, but I don't think there's anything that would prompt frequent re-renders that need to be optimized?